queue/redis/opts.go

```diff
@@ -132,6 +136,10 @@ func WithPublicKey(key string) ClientOpt {
 		c.config.PublicKey = new([32]byte)
 		copy(c.config.PublicKey[:], decoded)
 
+		if len(*c.config.PublicKey) != 32 {
+			return errors.New("no valid signing public key provided")
+		}
+
 		if c.config.PublicKey == nil {
 			return errors.New("unable to copy decoded queue signing public key, copied key is nil")
```

Comment:
```suggestion
			return errors.New("unable to copy decoded queue public key, copied key is nil")
```
to match with previous suggested change. otherwise it reads like public key is used for queue signing.